docs/system-tools/web-ui.md

```diff
@@ -148,11 +148,11 @@ The primary mechanism we recommend for debugging is [Temporal Web](https://githu
 
 ![6XkjmR](https://user-images.githubusercontent.com/6764957/110544958-71746480-8167-11eb-8152-8d3a3eb73d4e.gif)
 
-- [Workflows](https://docs.temporal.io/docs/glossary/#workflow) are identified by their [**Workflow ID**](https://docs.temporal.io/docs/glossary/#workflow-id), which you provide when creating the workflow. They also have a **Name** which is directly taken from your code.
+- [Workflows](https://docs.temporal.io/docs/reference/glossary/#workflow) are identified by their [**Workflow ID**](https://docs.temporal.io/docs/reference/glossary/#workflow-id), which you provide when creating the workflow. They also have a **Name** which is directly taken from your code.
```

Comment:
```suggestion
- [Workflows](/docs/content/what-is-a-workflow) are identified by their [**Workflow ID**](/docs/content/what-is-a-workflow-id), which you provide when creating the workflow. They also have a **Name** which is directly taken from your code.
```
Should the Workflows link instead be [Workflow Executions](/docs/content/what-is-a-workflow-execution)?

Reply:
Ya, this whole paragraph is full of ambiguity.